keras_retinanet/models/mobilenet.py

```diff
@@ -0,0 +1,56 @@
+import keras
+from keras import backend as K
+from keras.models import Model
+from keras.applications.mobilenet import MobileNet, BASE_WEIGHT_PATH, get_file
+
+from ..models import retinanet
+
+custom_objects = retinanet.custom_objects.copy()
+
+
+def download_imagenet(backbone):
+    alpha = 1.0
+    rows = backbone
+
+    # load weights
+    if K.image_data_format() == 'channels_first':
+        raise ValueError('Weights for "channels_last" format '
+                         'are not available.')
+    if alpha == 1.0:
+        alpha_text = '1_0'
+    elif alpha == 0.75:
+        alpha_text = '7_5'
+    elif alpha == 0.50:
+        alpha_text = '5_0'
+    else:
+        alpha_text = '2_5'
+
+    model_name = 'mobilenet_%s_%d_tf_no_top.h5' % (alpha_text, rows)
+    weigh_path = BASE_WEIGHT_PATH + model_name
+    weights_path = get_file(model_name,
+                            weigh_path,
+                            cache_subdir='models')
+
+    return weights_path
+
+
+def mobilenet_retinanet(num_classes, backbone=224, inputs=None, **kwargs):
+    alpha = 1.0
+
+    # choose default input
+    if inputs is None:
+        inputs = keras.layers.Input((None, None, 3))
+
+    mobilenet = MobileNet(input_tensor=inputs, alpha=alpha, include_top=False, pooling=None, weights=None)
+    outputs = [mobilenet.get_layer(name='conv_pw_{}_relu'.format(i)).output for i in [3, 5, 11, 13]]
+
+    # create the mobilenet backbone
+    mobilenet = Model(inputs=inputs, outputs=outputs, name=mobilenet.name)
+
+    # create the full model
+    model = retinanet.retinanet_bbox(inputs=inputs,
```

Comment:
I know it's not default PEP8, but I much more prefer the alternative style for multi-line function calling (which is also permitted by PEP8):
```python
foo(
    bar,
    baz=True
)
```

Reply:
Didnt really understand this. Can we have: 
foo(
    bar, bar2,
    baz=True,
    baz2=False
) ?

Reply:
As far as I'm concerned, yeah. Not sure what PEP8 says though.